lib/SIL/OperandOwnership.cpp

```diff
@@ -1033,6 +1033,12 @@ CONSTANT_OWNERSHIP_BUILTIN(Owned, MustBeInvalidated, UnsafeGuaranteed)
 #define BUILTIN(X, Y, Z)
 #include "swift/AST/Builtins.def"
 
+OperandOwnershipKindMap
+OperandOwnershipKindBuiltinClassifier::visitConvertStrongToUnownedUnsafe(
+    BuiltinInst *, StringRef) {
+  llvm_unreachable("Builtin should have been lowered to SIL instruction?!");
+}
```

Comment:
Wouldn't the BUILTIN_SIL_OPERATION expansion handle this already?

Reply:
I was following the example of GlobalStringTablePointer. Let me see.